skins/wmde19/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "wmde19",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:e2e": "vue-cli-service test:e2e",
+    "test:unit": "vue-cli-service test:unit"
```

Comment:
To make the current CI pass, you could add the following tasks:

```
"ci": "npm run lint && npm run test:unit && npm run test:e2e",
"build-assets": "npm run build"
```

Later, we'll integrate the skin better.